components/EmberPage.vue

```diff
@@ -195,8 +205,8 @@ export default {
       // If the iframe already exists, check if it is ready for us to reuse
       // by navigating within the app that is already loaded
       if (iframeEl !== null) {
-        const ready = iframeEl.getAttribute('data-ready') !== 'false';
-        const lastDidLoad = iframeEl.getAttribute('data-loaded') !== 'false';
+        const ready = this.getBooleanAttr(iframeEl, 'data-ready');
```

Comment:
Couldn't this just be `iframeEl.getAttribute('data-ready') === 'true'`?

Reply:
Yeah - there was a reason I didn't do that originally, but I can't see on reflection why that shouldn't be okay, so I'll swap it to use that

Reply:
Switched back to how it was but with explicit check for true